docs/boilerplate/app/context/Context.md

```diff
@@ -0,0 +1,24 @@
+# Context folder
+
+The `context` folder is where you can put your [React context providers](https://react.dev/learn/passing-data-deeply-with-context) or any other state management solutions you choose to use in your Ignited app.
+
+Ignite used to include [mobx-state-tree](https://mobx-state-tree.js.org/) as the default state management solution, but over time Infinite Red's clients have chosen to use it less and less. We still love MST, and it was our go-to solution for many years, but shipping it by default with Ignite makes less sense now that there are so many other great options out there. We haven't settled on a single state management solution so simple React contexts are the default in this boilerplate.
+
+Currently this folder contains the following React contexts that are meant to be simple examples of how to use React context in your app:
+
+- `AuthContext`: This is a basic context that provides the demo app with a way to manage simple authentication state. We persist the data using MMKV hooks in this context provider. The app can consume the AuthContext wih the `useAuth` hook.
+- `EpisodeContext`: This context provides the demo app with a way to manage the list of episodes in the demo podcast screen. This context provides a `useEpisodes` hook that can be used to fetch the episodes and manage the state of the list.
+
+## State management solutions
+
+There are many state management solutions available for React Native apps. React context comes built in and is a is a great solution for many apps. But you may want to consider other solutions depending on your app's complexity and needs.
```

Comment:
I'd rather not overstate the value of context ... it's fine, but I don't really think it's _great_ 😀 

```suggestion
There are many state management solutions available for React Native apps. React context comes built in and is a is a reasonable solution for many apps. But you may want to consider other solutions depending on your app's complexity and needs.
```